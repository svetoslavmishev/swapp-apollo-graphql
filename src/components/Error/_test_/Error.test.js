import React from 'react';
import { shallow } from 'enzyme';
import Error from '../Error';

describe('<Error />', () => {
  it('renders <Error /> component without error', () => {
    const wrapper = shallow(<Error />);
    expect(wrapper).toMatchSnapshot();
  });
});
