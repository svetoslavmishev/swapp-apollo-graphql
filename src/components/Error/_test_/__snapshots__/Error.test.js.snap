// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> renders <Error /> component without error 1`] = `ShallowWrapper {}`;
